### PR TITLE
LL-1615 Account migration flow

### PR DESCRIPTION
**Pull request by juan-cortes**
Group

### Type

Feature

### Context

https://ledgerhq.atlassian.net/browse/LL-1615

### CODE REVIEW

File: src/screens/MigrateAccounts/Banner.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+  if (!someAccountsNeedMigrationSelector) return null;
+
+  return (
+    <TouchableOpacity style={styles.root} onPress={navigateToAccountMigration}>
+      <View style={styles.logo}>
+        <LiveLogo size={16} color={colors.white} />
+      </View>
+      <LText semiBold style={styles.text}>
+        Ledger Live accounts update
```
Comment (MortalKastor) [inline src/screens/MigrateAccounts/Banner.js]: No _Actualización de cuentas de Ledger Live_ for you Spaniards :trollface:

Comment (juan-cortes) [reply src/screens/MigrateAccounts/Banner.js]: Tiene buen ojo, gracias

**Comment by juan-cortes** [comment]
@Arnaud97234 current version should work, I broke it when trying to make flow happy 🤷‍♂

**Comment by juan-cortes** [comment]
For this one @gre do we need to update the libcore/common version in order to fix what we had to do on desktop for ETC?

**Comment by gre** [comment]
@juan-cortes a rebase should do it

**Comment by gre** [comment]
@juan-cortes this can now be rebased as we have latest live-common in develop

**Comment by juan-cortes** [comment]
> @juan-cortes this can now be rebased as we have latest live-common in develop

And so it is now rebased

**Pull request merged**